docs/medical-api/getting-started/quickstart.mdx

```diff
@@ -219,3 +219,17 @@ That's it! You're now well equiped to use the Metriport API to access and manage
 
 As next steps, we'd recommend scoping out our [Medical API Reference](/medical-api/api-reference/organization/create-organization)
 in further detail to see the other things you can do with the API.
+
+## 8. Get Patient Resources
```

Comment:
- need to update the overview with step 8
- also perhaps title this `Get Patient FHIR Resources`?